rasgotransforms/rasgotransforms/row_transforms/sample/sample.yaml

```diff
@@ -0,0 +1,15 @@
+name: sample
+tags:
+  - row
+  - math
+  - conditional
+description: Take a sample of a dataset using a specific number of rows or a probability that each row will be selected
+arguments:
+  num_rows:
+    type: value
+    description: To sample using a probability of selecting each row, your num_rows should be a decimal less than 1. Otherwise, pass an integer value for number of rows to keep.
+example_code: |
+  ds = rasgo.get.dataset(id)
+
+  ds2 = ds.sample(sample_amount=1000)
```

Comment:
should `sample_amount` be `num_rows` here?

Reply:
thanks for catching